projects/plugins/jetpack/_inc/client/discussion/comments.jsx

```diff
@@ -193,7 +197,7 @@ class CommentsComponent extends React.Component {
 										toggleModule={ this.props.toggleModuleNow }
 									>
 										<span className="jp-form-toggle-explanation">
-											{ __( 'Enable comment likes.', 'jetpack' ) }
+											{ __( 'Enable comment Likes.', 'jetpack' ) }
```

Comment:
Nothing in this PR says anything about why this is changing?

Reply:
True. I've edited the PR description to mention it. Basically it was mentioned in the Slack thread that reported the original issue, and I figured I'd make that change in the same PR since it's a really tiny change.

Reply:
Works for me.